l10n/bundle.l10n.json

```diff
@@ -0,0 +1,5 @@
+{
+  "Edit the site": "Edit the site",
+  "Be careful making changes. Anyone can see the changes you make immediately. Choose Edit the site to make edits, or close the editor tab to cancel without editing.": "Be careful making changes. Anyone can see the changes you make immediately. Choose Edit the site to make edits, or close the editor tab to cancel without editing.",
```

Comment:
One gap that exist in the new localization file is that there was no way to introduce a key:sting mapping for better readability. This is something we can investigate further, and follow up on with VSCode team if needed.